### ISSUE TITLE
Print - legend (~?h)

### ISSUE THREAD

**Issue by anderserla** [Sundsvallskommun/OpenEMap-WebUserInterface#43]
Make it possible to include legend when printing.

**Comment by sweco-sebhar** [comment]
I don't think this is possible in mapfish directly. To do it we probably would need to write our own "legend-service" which can combine all input legend images from sources to one image. This in a non-trivial task.

**Comment by sweco-sebhar** [comment]
I don't think this is possible in mapfish directly. To do it we probably would need to write our own "legend-service" which can combine all input legend images from sources to one image. This in a non-trivial task, I guess more than one day will be needed. Needs further analysis before proper estimation, leaving it to @anderserla to decide on how to proceed.

**Comment by anderserla** [comment]
After discussion we ended up with different solutions:
1. Develop Mapfish so that we can either send in an image link as a predefined legend for a layer or let Mapfish generate a legend using standard WMS GetLegend call. (2 h investigation to estimate)
2. Make a right mouse button menu "print", that appears when right clicking the legend in the layer control. The menu item should open a web page containing only the selected legend and open the built in print dialog. It would also be great to have a "Copy" item in the right mouse buton menu. (4h)
3. Create predefined printable legends for the most important layers in the map, accesible from outside of Open eMap (?h)
